CMakeLists.txt

```diff
@@ -14,6 +14,8 @@ and Nvidia GPUs for hardware encoding. Software encoding is also available. You
 Moonlight client on a variety of devices. A web UI is provided to allow configuration, and client pairing, from \
 your favorite web browser. Pair from the local server or any mobile device.")
 
+include(GNUInstallDirs)
```

Comment:
Can this be put into a linux specific cmake file?

Reply:
Not an expert here but according to https://code-review.googlesource.com/c/re2/+/43990/5#message-36d7112c2c43973dcb63c5160937fb1981702010 which I found while googling for it that should be okay for other OS'es as well and I personally I didn't come across any special handling for GNUinstallDirs yet. Current re2 also has still has that in their main MakeLists.txt (https://github.com/google/re2/blob/main/CMakeLists.txt#L11) as well as cross-platform stuff like SDL2 (https://github.com/libsdl-org/SDL/blob/main/CMakeLists.txt#L39).

Reply:
> should be okay for other OS'es as well

That wasn't the reason for my request. It's more to keep the code separated. I went through a great effort trying to separate out the former monster of a CMakeLists.txt and don't want to head back in that direction if it can be avoided.

Reply:
Please advice in which file you'd like to see it moved then.

Reply:
Probably at the top of `./cmake/packaging/unix.cmake`